package/python-kiwi-spec-template

```diff
@@ -187,7 +187,9 @@ Recommends:     gfxboot
 Requires:       grub2-efi-x64
 %endif
 %endif
+%ifarch %{ix86} x86_64
 Requires:       grub2
+%endif
```

Comment:
This breaks it for non-x86 RPM distributions, who have their non-x86 variants of GRUB packages provide the `grub2` name.

Reply:
ok, so I changed it to have this requirement only for the distros that uses this concept. How about it now ?